src/ducks/upload/styles.styl

```diff
@@ -31,13 +31,17 @@
     margin           0
     padding          em(8px) em(16px) em(8px)
 
+    > .upload-queue-header-inner
```

Comment:
now I'm questioning the need for the children selector `>`. It did have a purpose when you used a `div` instead of a class but now… it just give more unnecessary weight to the selector.